payloads/external/edk2/Kconfig

```diff
@@ -248,6 +248,40 @@ config EDK2_OPAL_PASSWORD
 	help
 	  Enables TCG OPAL disk password in the TianoCore Payload.
 
+# DasharoModulePkg specific options
+
+config EDK2_DASHARO_SECURITY_OPTIONS
+	bool "Enable EDK2 Dasharo Security Options Menu"
+	default n
+	help
+	  Enables the PS/2 keyboard configuration submenu in Dasharo System Features.
```

Comment:
Security options and PS/2? Copy pasta?

We must take care of the lock case, so that the platform won't be locked if menu is hidden, otherwise the lock would be impossible to be disabled...

Reply:
Copy / paste error fixed.